src/lib/application.ts

```diff
@@ -272,7 +272,7 @@ export class Application extends ChildableComponent<Application, AbstractCompone
             file = Path.resolve(file);
             if (FS.statSync(file).isDirectory()) {
                 add(file);
-            } else {
+            } else if (exclude && !exclude.match(file)) {
```

Comment:
Just wanted to note that #395 suggested there was a problem here. Thanks for the fix!